cypress-io/cypress
# fix: Allow typing seconds into datetime inputs

- Closes #22884

### User facing changelog

`cy.type()` can now be used to enter datetime values with seconds, to an input of type `datetime-local`, for example `1959-09-13T10:12:13`. 

### Additional details


Seconds are valid in this datetime format, our regex was overly narrow.

### Steps to test


Run `packages/driver/cypress/e2e/commands/actions/type.cy.js` and see that the new tests are passing.

Run the test snippet provided in #22884 and see that it also passes now.

### How has the user experience changed?


n/a

### PR Tasks


- [na] Has a PR for user-facing changes been opened in [`cypress-documentation`](https://github.com/cypress-io/cypress-documentation)? 
- [na] Have API changes been updated in the [`type definitions`](https://github.com/cypress-io/cypress/blob/develop/cli/types/cypress.d.ts)?
